File: java/benchmark_2.java
import java.util.Date;

class benchmark_2  {
    public static void main(String[] args) {
      String x="";
      long start = new Date().getTime();
      for (int i = 0; i < 100000; i++) {
        x = x + "x";
      }
      long end = new Date().getTime();
      long total_time = end-start;
      System.out.println("Total time=" + total_time+"ms");
    }
}

File: python/benchmark_2.py
#!/usr/bin/python

from datetime import datetime
 
x=""
start = datetime.now();

i = 0
while i < 100000:
  i = i + 1
  x = x + "x"
  
end = datetime.now();
total_time = (end.microsecond-start.microsecond)/1000
print "Total time="+str(total_time)+"ms"